.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -16,6 +16,14 @@ If you've checked more than one of the first three boxes, consider splitting thi
 - [ ] `Testing`: Adds testing
 - [ ] `Docs`: Adds documentation
 
+## Versioning Implications
+
+- [ ] This PR will affect [semantic versioning as defined for ICS](../CONTRIBUTING.md#branching-model-and-release)
+
+New ICS semantic version: `X.Y.Z`
+
+- [ ] Has the new version been included in a tag?
```

Comment:
This would likely be done after the PR is merged, so maybe not relevant. Maybe there's some automation possible here

Reply:
"Has the new version been included in a tag?"
what does this mean ?😄

Reply:
Meaning a commit has been tagged with the bumped version